docs/react-v9/contributing/command-cheat-sheet.md

```diff
@@ -1,4 +1,30 @@
-// TODO #30642 A doc showing a table of every day commands.
+This is a helpful list of helpful every day commands.
 
-Assignee - Mason
-Reviewer - Esteban
+yarn // installs everything. It fixes many things.
+yarn build // builds the whole monorepo. You should only need to do this on initial startup.
```

Comment:
>  You should only need to do this on initial startup.

that's not true,

- all v9 workflows work without need to build ( besides generating api.md ) files

- for v8 you need to build in order to make TS intelisense work, but that's necessary only via `lage build --to <package-name>` usually `lage build --to react`

Reply:
okay - will add that sequence to his doc at the bottom.